spec/formatting.md

```diff
@@ -222,6 +222,11 @@ the following steps are taken:
 
    The form that resolved _operand_ and _option_ values take is implementation-defined.
 
+   Since the result of a function call can be bound to a _variable_,
+   the output of one _function_ may be the input of another _function_.
```

Comment:
I made a slightly different change instead (in order to avoid implying that an expression has output, which isn't really a concept in the spec); let me know what you think.